### PR TITLE
support ghc 810

### LINKED ISSUE #75: GHC 8.10 support?
I noticed that you don't support GHC 8.10. Could you?

### PR DESCRIPTION
**Pull request by mankyKitty**
... I think

Fixes #75

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Revision history for waargonaut
 
+## 0.8.0.2  -- 2020-06-02
+
+- Support GHC 8.10.0
```
Comment (gwils) [inline changelog.md:5]: technically 8.10.1

**Comment by gwils** [comment]
It's also worth updating the travis config. Add `8.10.1` to the list of `tested-with` and then

```
cabal update
cabal install haskell-ci
haskell-ci waargonaut.cabal
```

**Comment by gwils** [comment]
You may also need to resolve https://github.com/system-f/natural/issues/5

**Comment by mankyKitty** [comment]
https://github.com/qfpl/hs-hoist-error needs upgrade to support new GHC

**Comment by kozross** [comment]
What's the status of this, out of interest?

**Comment by mankyKitty** [comment]
Current status is well overdue, but it is happening, just difficult to prioritise because \***gestures at everything**\*.

**Comment by emilypi** [comment]
Happy to take this over :)

**Comment by emilypi** [comment]
An awful lot has happened in 6 months! I'm picking this up and I'll have it working within the next few days. Since the last time we checked in, Travis decided to nuke its OSS support, so I'll migrate us to Github actions.

File: changelog.md (hunk 1 above)
Comment (emilypi) [reply changelog.md]: Bumped in latest commits

**Comment by emilypi** [comment]
@gwils @mankyKitty do you mind giving a look to my latest commits? Looks like there's a bit of Nix work to be done, but unfortunately I upgraded to Big Sur without thinking and static linking for Nix is badly broken without alot of work on my part. Otherwise, it seems to build

File: waarg-overrides.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
+sources: hlib: hself: hsuper: {
+  # doctest = hsuper.callCabal2nix "doctest" sources.doctest {};
+  # haskellworks
+  hw-bits = hsuper.callCabal2nix "hw-bits" sources.hw-bits {};
+  hw-balancedparens = hlib.dontCheck (
+    hsuper.callCabal2nix "hw-balancedparens" sources.hw-balancedparens {}
+  );
+  hw-json-simd = hlib.markUnbroken hsuper.hw-json-simd;
```
Comment (emilypi) [inline waarg-overrides.nix:8]: amended: `hw-json-simd` is marekd as broken downstream. I've marked it unbroken here.

**Comment by gwils** [comment]
When I try to `nix-shell`, I get
```
Test suite hw-json-simd-test: RUNNING...
Test suite hw-json-simd-test: PASS
Test suite logged to: dist/test/hw-json-simd-0.1.1.0-hw-json-simd-test.log
Test suite doctest: RUNNING...

/build/hw-json-simd-0.1.1.0/src/HaskellWorks/Data/Json/Simd/Capabilities.hs:3:1: error:
    Could not find module ‘HaskellWorks.Data.Json.Simd.Internal.Foreign’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
  |
3 | import qualified HaskellWorks.Data.Json.Simd.Internal.Foreign as F
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Test suite doctest: FAIL
Test suite logged to: dist/test/hw-json-simd-0.1.1.0-doctest.log
1 of 2 test suites (1 of 2 test cases) passed.
builder for '/nix/store/810sl1sclnfpnfhh5b49jghcifds8vv4-hw-json-simd-0.1.1.0.drv' failed with exit code 1
cannot build derivation '/nix/store/41wz9q7if2iygac48azx6di4i7xk1hgl-ghc-8.8.4-with-packages.drv': 1 dependencies couldn't be built
error: build of '/nix/store/41wz9q7if2iygac48azx6di4i7xk1hgl-ghc-8.8.4-with-packages.drv' failed
```

**Comment by emilypi** [comment]
Alright, I can't get this to work for nix. However, I'm no going to let it block release. Merging, and I'll take out an issue for fixing the nix build ASAP.

**Pull request merged**